modules/@shopify/checkout-sheet-kit/ios/ShopifyCheckoutSheetKit.swift

```diff
@@ -182,6 +182,10 @@ class RCTShopifyCheckoutSheetKit: RCTEventEmitter, CheckoutDelegate {
 		}
 	}
 
+  @objc func invalidateCache() {
+		ShopifyCheckoutSheetKit.invalidate()
+	}
```

Comment:
`invalidate` is an existing method in React-Native land so I've opted to call it this instead